.github/workflows/mergedtomain-workflow.yaml

```diff
@@ -16,14 +16,65 @@ jobs:
       # write permission is required to create a github release
       contents: write
       pull-requests: write
+      id-token: write  # needed for cosign keyless signing with OIDC
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get latest release
+        id: get_latest_release
+        run: |
+          echo "release=$(git branch -r | grep releases\/v | sort -V | tail -1 |  awk -F'/' '{ print $3 }')" >> $GITHUB_OUTPUT
+
+      - name: 'Get next patch version'
+        id: semvers
+        uses: "WyriHaximus/github-action-next-semvers@18aa9ed4152808ab99b88d71f5481e41f8d89930"
+        with:
+          version: ${{ steps.get_latest_release.outputs.release }}
```

Comment:
How about to use something like https://github.com/mdomke/git-semver - it gives us nice semvers?